### ISSUE TITLE
Implement 3.8 scheme editing features

### ISSUE THREAD

**Issue by AdameBf** [AdameBf/wawse#94]
W:A version 3.8 will feature a new scheme format and numerous new scheme settings. These should be handled.

**Comment by ghost** [comment]
Any open documentation?

**Comment by AdameBf** [comment]
> Any open documentation?

Yup, check https://worms2d.info/Game_scheme_file and the *Extended options* section in particular!

**Comment by skunkmommy** [comment]
Bump
